src/params/mod.rs

```diff
@@ -105,6 +104,9 @@ pub enum HashChoice {
     Blake2s,
     /// The BLAKE2b hash function, designed to be more efficient on 64-bit architectures.
     Blake2b,
+    /// The BLAKE3 hash function, designed to be more efficient than BLAKE3 through,
+    /// in part, parallelism.
+    Blake3,
```

Comment:
`snow` has a policy of feature-gating any non-spec cryptographic primitives (including their enum variants), such that the default build is always specification compliant. So we'll want this setup in a way similar to `P-256` and `XChaCha20Poly1305`, and make sure if `use-blake3` isn't specified, there will be nothing mentioning BLAKE3 compiled in `snow`.

Reply:
Of course, overlooked the precedent set by XChaCha20-Poly1305